acsylla/_cython/result/row.pyx

```diff
@@ -17,25 +17,42 @@ cdef class Row:
 
         return row
 
+    def __iter__(self):
+        return self.as_named_tuple()
+
     def column_count(self):
         return self.result.column_count()
 
     def as_dict(self):
-        cdef Py_ssize_t length = 0
-        cdef char* name = NULL
-        cdef CassError error
-        ret = {}
-        for i in range(self.result.column_count()):
-            error = cass_result_column_name(self.result.cass_result, i, <const char**> &name, <size_t*> &length)
-            raise_if_error(error)
-            ret[name.decode()] = self.column_value(name.decode())
```

Comment:
I would say that here we were leaking some memory. For the same that was commented above.